src/api/api.tsx

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+
+import {GoodsData,ThemeData } from '@/types';
+
+const API_URL = axios.create({
```

Comment:
`API_URL` 보다는 다른 네이밍이 적확한거 같습니다.

```suggestion
const axiosInstance = axios.create({
```